src/content/developers/docs/smart-contracts/formal-verification.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Formal verification of smart contracts
+description: An overview of formal verification techniques for Ethereum smart contracts 
+lang: en
+sidebar: true
+---
+
+
+## A brief introduction {#introduction-to-formal-verification}
```

Comment:
I'd say we could just remove this header. Many of our docs pages jump straight into the intro, e.g.
- https://ethereum.org/en/developers/docs/smart-contracts/languages/
- https://ethereum.org/en/developers/docs/smart-contracts/anatomy/
- https://ethereum.org/en/developers/docs/smart-contracts/libraries/